### ISSUE TITLE
CountUp starts over on re-renders

### ISSUE THREAD

**Issue by kershnerd** [glennreyes/react-countup#540]
I have a functional component that draws a chart and overlays a CountUp on top.

```js
<CountUp className="kpi-text" start={0} end={value} suffix={props.suffix} formattingFn={formatGrouping} duration={4} />
```

How can I make the `CountUp` only render if the start or end value have changed?

**Comment by tpudel** [comment]
Having the same issue, have you solved this yet?
Started happening since the major upgrade from version 4 to 5 for me

**Comment by kershnerd** [comment]
Even though my props didn't change, my component re-rendered higher up the tree which caused my component that had the `CountUp` to re-animate.

I wasn't able to fix this directly but using a memoization seemed to work.

Import  memo from react:
```
import React, { useEffect, memo } from 'react';
```

At the bottom when you export your component:
```
export default memo(AveragePeopleInLine);
```

This will force the component to not re-render if the props have not changed.

**Comment by tpudel** [comment]
> @tpudel could you please provide a small representation?

Sure. A simple example would be this:
https://codesandbox.io/s/modest-fast-l9e56?file=/src/App.js

First time using code sandbox, let me know if you cannot see it!

**Comment by mmarkelov** [comment]
Thanks guys! I could reproduce it and already fixed this. It should be fine in **6.0.0-1**. And I think that i'll publish this as stable release soon

**Comment by tpudel** [comment]
Do you happen to have a ETA on the next major stable release?

**Comment by mmarkelov** [comment]
@tpudel just released **6.0.0** with this fix

**Comment by shubhamtarpara** [comment]
Hey guys, whenever my component gets re-render at a higher level animation start again and again how to stop that with a memo its works but is there any other solution?
